ui/src/pipeline/pipeline.consumer.pact.test.ts

```diff
@@ -150,8 +153,7 @@ pactWith(options, provider => {
       });
     });
 
-    // TODO do not skip these tests anymore as soon as issue #353 is solved
-    describe.skip('getting a pipeline by datasource id', () => {
+    describe('getting pipelines by datasource id', () => {
       describe('when a pipeline with the requested datasource id exist', () => {
```

Comment:
I noticed a typo here, I guess that was my bad:
```suggestion
      describe('when a pipeline with the requested datasource id exists', () => {
```